app/src/main/res/values/strings.xml

```diff
@@ -1685,4 +1685,8 @@
     <string name="onboarding_firefox_account_auto_signin_header_2">You are signed in as %s on another Firefox browser on this phone. Would you like to sign in with this account?</string>
     <!-- Deprecated: Describes the add to homescreen functionality -->
     <string name="add_to_homescreen_description">You can easily add this website to your phone’s Home screen to have instant access and browse faster with an app-like experience.</string>
+    <!--Content Description for webAddressView -->
```

Comment:
We also try to group the strings close to their related peers. Since this relates to logins, you can find more login related strings around https://github.com/mozilla-mobile/fenix/blob/cc0fdd1109d1ac7075505363dd737fc9af05811d/app/src/main/res/values/strings.xml#L1507. I would move these strings there and add an appropriate string prefix depending on the screen (eg `saved_logins` or `logins`).

Reply:
Done